docs/python_api/datasource/gwas_catalog/_gwas_catalog.md

```diff
@@ -6,3 +6,17 @@ title: GWAS Catalog
   <img width="100" height="100" src="../../../../assets/imgs/GWAS_Catalog_circle_178x178.png">
   <h1>GWAS Catalog</h1>
 </div>
+
+The [GWAS Catalog](https://www.ebi.ac.uk/gwas/) is a comprehensive resource that aims to provide a curated collection of Genome-Wide Association Studies (GWAS) across various traits and diseases in humans.
+
```

Comment:
I would also add harmonised summary statistics as an ingested piece of data that they provide.

Reply:
I think it is included into the curated collection. But I added the phrase.